numpyro/contrib/distributions/__init__.py

```diff
@@ -47,3 +51,50 @@
     'StudentT',
     'Uniform',
 ]
+
+# TODO: remove before release
+
+
+def get_bern(p, is_logits=False):
+    if is_logits:
+        return BernoulliWithLogits(p)
+    else:
+        return Bernoulli(p)
+
+
+def get_binom(n, p, is_logits=False):
+    if is_logits:
+        return BinomialWithLogits(p, n)
+    else:
+        return Binomial(p, n)
+
+
+def get_categorical(p, is_logits=False):
+    if is_logits:
+        return CategoricalWithLogits(p)
+    else:
+        return Categorical(p)
+
+
+def get_multinomial(n, p, is_logits=False):
+    if is_logits:
+        return MultinomialWithLogits(p, n)
+    else:
+        return MultinomialWithLogits(p, n)
+
+
+beta = Beta
+bernoulli = get_bern
+binom = get_binom
+cauchy = lambda loc=0., scale=1.: Cauchy(loc, scale)
+categorical = get_categorical
+dirichlet = lambda alpha: Dirichlet(concentration=alpha)
+expon = Exponential
+gamma = lambda conc, scale=1.: Gamma(conc, rate=1./scale)
+halfcauchy = lambda scale: HalfCauchy(scale)
+lognorm = lambda s, scale=1.: LogNormal(loc=np.log(scale), scale=s)
+multinomial = get_multinomial
+norm = lambda loc=0., scale=1.: Normal(loc, scale)
+pareto = lambda alpha, scale=1.: Pareto(scale, alpha)
+t = lambda df, loc=0., scale=1.: StudentT(df, loc, scale)
+uniform = Uniform
```

Comment:
Why don't we just copy the files over the new places instead of making wrappers like this?

Reply:
I am just doing this in phases. Once this PR merges, I will have a simple PR that moves around the files and changes the names etc. Didn't want to put in too many changes in one PR.